app/models/form_profiles/va_686c674v2.rb

```diff
@@ -37,11 +37,7 @@ def metadata
 
   def prefill_form_address
     begin
-      mailing_address = if user.icn.present? && Flipper.enabled?(:remove_pciu, user)
-                          VAProfileRedis::V2::ContactInformation.for_user(user).mailing_address
-                        elsif user.vet360_id.present?
-                          VAProfileRedis::ContactInformation.for_user(user).mailing_address
-                        end
+      mailing_address = VAProfileRedis::V2::ContactInformation.for_user(user).mailing_address
```

Comment:
@RachalCassity do you still want to check for the ICN?

Reply:
Rendering `VAProfileRedis::V2::ContactInformation` will trigger a check for the ICN.